frontend/components/player/now_playing_info.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect} from "react";
+
+const NowPlayingInfo = ({
+	audioRef,
+	track,
```

Comment:
Adding a default `track` prop here or as NowPlayingInfo.defaultProps before `export default` were both not working.